frontend/test/__runner__/cypress-runner-backend.js

```diff
@@ -106,9 +104,29 @@ const CypressBackend = {
       return new Promise((resolve, reject) => setTimeout(resolve, duration));
     }
 
+    async function promXhr(uri, options) {
+      const xhr = new XMLHttpRequest();
+      return new Promise((resolve, reject) => {
+        xhr.open(options.method, uri, true);
+        for (const headerName in options.headers) {
+          xhr.setRequestHeader(headerName, options.headers[headerName]);
+        }
+        xhr.send(options.body);
+        xhr.onload = () => {
+          const responseObject = {
+            status: xhr.status,
+            response: xhr.responseText,
+          };
+          xhr.status >= 200 && xhr.status <= 299
+            ? resolve(responseObject)
+            : reject(responseObject);
+        };
+      });
+    }
+
     async function isReady(host) {
       try {
-        const { status } = await fetch(`${host}/api/health`);
+        const { status } = await promXhr(`${host}/api/health`);
```

Comment:
This doesn't work because `promXhr` expects `options` with a `method` property on it (and method here is undefined).

1. why is the order or parameters in `promXhr` reversed from the arguments used to invoke `xhr.open`?
2. btw, even when I update the call to properly include the method like this 
```js
const { status } = await promXhr(`${host}/api/health`, { method: "GET" });
```
it still never returns any value.

Reply:
just tried with the change you proposed and it works in my console: 
![image](https://user-images.githubusercontent.com/1711649/195875772-5df2f7c2-5a90-4bee-b1c5-2fbf422418e5.png)

Reply:
BTW: sorry for missing that argument, very very basic mistake

Reply:
It happens.
Cypress backend doesn't need this full-blown implementation because it's only ever going to use GET and only for one route.

But, it's still insane that it doesn't work in CI.
I also cannot get it to work locally.